content/en/docs/concepts/services-networking/service.md

```diff
@@ -619,6 +619,77 @@ You can integrate with [Gateway](https://gateway-api.sigs.k8s.io/) rather than S
+by load balancers to determine which VM instance the load balancer should
+dispatch traffic on. The health check probe in this context is not a Kubernetes
+health check probe (as in liveness/readiness) but a construct cloud providers
+expose with the goal of allowing users the possibility to impact traffic load
+balancing. Kubernetes uses this within the context of supporting the feature
+`externalTrafficPolicy`. If `cluster` is specified all nodes are eligible load
+balancing targets _as long as_ the node is not being deleted and Kube-proxy is
+healthy. In this mode: Kubernetes configures the load balancer health check to
+target the service proxy's readiness port and path. In the case of Kube-proxy
+this evaluates to: `${NODE_IP}:10256/healthz`. Kube-proxy will return either an
+HTTP code 200 or 503. It will return 200 if a) Kube-proxy is healthy, meaning:
+it's able to progress programming the network and isn't timing out while doing
+so (the timeout is defined to be: `2 *
+kubeProxyIPTablesConfiguration.syncPeriod`) and b) the Node is not being
+deleted. The reason why Kube-proxy returns 503 and marks the node as not
+eligible when it's being deleted, is because Kube-proxy supports connection
+draining for terminating nodes. A couple important things occur from the point
+of view of a Kubernetes load balancer when a node _is being_ / _is_ deleted.
+
+While deleting:
+
+* Kube-proxy will start failing its readiness probe and essentially mark the
+   node as not eligible for load balancer traffic. The load balancer health
+   check failing causes load balancers which support connection draining to
+   allow existing connections to terminate, and block new connections from
+   establishing.
+
+When deleted:
+
+* The service controller in the Kubernetes cloud controller manager removes the
+  node from the referenced set of eligible targets. Removing any instance from
+  the load balancer's set of backend targets immediately terminates all
+  connections. This is also the reason Kube-proxy first fails the health check
+  while the node is deleting.
+
+It's important to note for Kubernetes vendors that if any vendor configures the
+Kube-proxy readiness probe as a liveness probe: that Kube-proxy will start
+restarting continuously when a node is deleting until it has been fully deleted.
+
+Users deploying Kube-proxy can inspect both the readiness / liveness state by
+evaluating the metrics: `proxy_livez_total` / `proxy_healthz_total`. Both
+metrics publish two series, one with the 200 label and one with the 503 one.
+
+For services of `externalTrafficPolicy: local`:  Kube-proxy will return 200 if
+a) Kube-proxy is healthy/ready and b) has a local endpoint on the node in
+question. Note: node deletion does not have an impact on Kube-proxy's return
+code for what concerns load balancer health checks. The reason for this is:
+deleting nodes could end up causing an ingress outage should all endpoints
+simultaneously be running on said nodes.
```

Comment:
```suggestion
1. kube-proxy is healthy/ready, and
2. kube-proxy has a local endpoint on the node in question.  
   Node deletion does **not** have an impact on kube-proxy's return code for what
   concerns load balancer health checks. This is because deleting nodes could end up causing
   an ingress outage should all endpoints simultaneously be running on said nodes.
```